example/src/nested-js/index.js

```diff
@@ -0,0 +1,21 @@
+import Component from '../../../src/index';
+import { TextInput } from './child';
+
+class Form extends Component {
+
+
+  render() {
+    this.html`
+      <form>
+        <x-text-input />
+      </form>
+    `;
+  }
+}
+
+Form.componentName = 'x-form';
+Form.dependencies = [
```

Comment:
Is there the possibility to have these 2 lines as static? If not, then they're fine as is.

Reply:
I tested it 👍 nope, just static methods in classes are allowed.